docker/flags.go

```diff
@@ -50,7 +50,7 @@ var (
 		{"logs", "Fetch the logs of a container"},
 		{"port", "Lookup the public-facing port that is NAT-ed to PRIVATE_PORT"},
 		{"pause", "Pause all processes within a container"},
-		{"ps", "List containers"},
+		{"ps", "List running containers"},
```

Comment:
I think `List containers` is correct, because `docker ps` with different option can list containers with different  state,
such as `docker ps -a` is to list all the containers, `docker ps -l` is list last created containers ... etc
`docker ps` with no option is to list running containers